mindsdb/utilities/fs.py

```diff
@@ -124,14 +124,18 @@ def clean_unlinked_process_marks():
                     next(t for t in threads if t.id == thread_id)
                 except StopIteration:
                     from mindsdb.utilities.log import get_log
-                    get_log('main').warning(
-                        f'We have mark for process/thread {process_id}/{thread_id} but it does not exists'
+
+                    get_log("main").warning(
+                        f"We have mark for process/thread {process_id}/{thread_id} but it does not exists"
                     )
                     file.unlink()
+            except psutil.AccessDenied:
```

Comment:
This gets rid of the psutil error I mentioned on the ML call on Monday

Reply:
@StpMax @ea-rus is it okay to ignore processes and/or threads that can't be accessed? Seems like a rare edge case. This seems like the only possible course of action when unmarking. Instead, we may want to prevent creating processes that won't be accessible later, but I'm not super clear on how this bit of the codebase works. Please let us know if this is a valid handling of the `AccessDenied` exception.